src/Hedgehog/Property.fs

```diff
@@ -270,14 +287,19 @@ module Property =
             | Success x ->
                 Gen.map (first (Journal.append journal)) (k x)
 
+    [<CompiledName("Bind")>]
     let bind (m : Property<'a>) (k : 'a -> Property<'b>) : Property<'b> =
         bindGen (toGen m) (toGen << k) |> ofGen
 
+    [<CompiledName("ForAll")>]
     let forAll (gen : Gen<'a>) (k : 'a -> Property<'b>) : Property<'b> =
         let prepend (x : 'a) =
             bind (counterexample (sprintf "%A" x)) (fun _ -> k x) |> toGen
         Gen.bind gen prepend |> ofGen
 
+    [<CompiledName("ForAll")>]
+    let forAll' (gen : Gen<'a>) : Property<'a> = forAll gen success
```

Comment:
Nitpick^2:

```f#
let forAll' (gen : Gen<'a>) : Property<'a> = forAll gen success
```

might as well be written as

```f#
let forAll' (gen : Gen<'a>) : Property<'a> =
    forAll gen success
```

so that it's consistent with the rest of the function/member declarations in the codebase. I think it's the same in Haskell version too. /cc @jystic